src/main/java/com/amadeus/resources/HotelRating.java

```diff
@@ -0,0 +1,29 @@
+package com.amadeus.resources;
+
+import lombok.Getter;
+
+public class HotelRating extends Resource {
```

Comment:
The class should be named HotelSentiment as it is the object returned by the API:

![image](https://user-images.githubusercontent.com/713173/67024835-5b46ae00-f105-11e9-87ff-d7e7db849d44.png)

![image](https://user-images.githubusercontent.com/713173/67024874-68639d00-f105-11e9-97e6-3173ed7c4439.png)

I fully agree: this is confusing. We changed the "commercial" name of the API to be Hotel Ratings as we thought it was more meaningful for the users but the product itself was named Hotel Sentiments...